helm-deploy-test/helm-deploy-test.yml

```diff
@@ -36,7 +36,7 @@ resources:
     pool: ((kube-pool-pool))
 
 jobs:
-- name: helm-deploy-test-openSUSE
+- name: helm-deploy-sa-test-openSUSE
```

Comment:
did u mean "ha", rather than "sa-"

Reply:
I guess we should either remove non-ha opensuse or ha-opensuse jobs, we should not use 2 resources(caasp) to test opensue. 1 out of two should be enough. I guess we should keep HA and remove nonHA opensuse job

Reply:
sa if for single/standard availability